corehq/apps/ota/views.py

```diff
@@ -157,8 +157,14 @@ def claim(request, domain):
                    host_name=unquote(request.POST.get('case_name', '')),
                    device_id=__name__ + ".claim")
 
-    if set(case_ids) == case_ids_already_claimed:
-        return HttpResponse(status=204)
+    if not case_ids_to_claim:
+        from casexml.apps.phone.models import get_properly_wrapped_sync_log
```

Comment:
Is the inline import necessary?

Reply:
When should we import inline vs. at the top?

Reply:
Imports should always be at the top unless there's a specific reason not to